client/src/store/modules/list.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios'
+
+export default {
+  actions: {
+    getLikedList (ctx, videoId) {
+      axios.get(`/api/posts/get-post-likes/${videoId}`)
```

Comment:
Don't use http requests inside actions, would be nicer to have API service and simply import needed calls from there.

Reply:
**Something like this?**
```
import postService from '../../services/post'
export default {
  actions: {
    getLikedList (ctx, videoId) {
      ctx.commit('updateLikedList', postService.getLikedList(videoId))
    }
  },
```

Reply:
async getLikedList (ctx, videoId) {
     const likedList = await postService.getLikedList(videoId);
      ctx.commit('updateLikedList', likedList)
    }

Reply:
Something similar that you posted with a few tweaks.